src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdClientSettings.Edit.cshtml

```diff
@@ -13,209 +14,210 @@
     <label asp-for="DisplayName" class="form-label">@T["Display Name"]</label>
     <input asp-for="DisplayName" class="form-control" />
     <span asp-validation-for="DisplayName"></span>
-    <span class="hint">@T["Display name of the IdP"]</span>
+    <span class="hint">@T["User-Friendly Display Name of the Identity Provider"]</span>
```

Comment:
I'd personally avoid using upper-casing in descriptions or hints.

Reply:
"Display Name" and "Identity Provider" are nouns and why they are capitalized in this case.